### PR TITLE
Update lamarzocco documentation with connection methods per entity

### PR DESCRIPTION
**Pull request by zweckj**
## Proposed change

- document connection methdod per entity
- backtick models for readability

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Added Bluetooth support for automatic discovery of machines.
	- Introduced a configuration option to manually disable Bluetooth communication.

- **Documentation**
	- Clarified update frequency for machine state (30 seconds), statistics (5 minutes), and firmware checks (1 hour).
	- Updated tables with new icons for Bluetooth and cloud connectivity.
	- Enhanced troubleshooting guidance for connection issues, including verification through the La Marzocco Home app and proper configuration of the host option.

### CODE REVIEW

File: source/_integrations/lamarzocco.markdown [hunk 1]
```diff
@@ -90,19 +90,19 @@ Use Bluetooth:
...
 ## Switches
 
-| Switch name | Description | Available for machines |
-|-------------|-------------| ---------------------- |
-| Main        | Allows to turn machines on-/off | all |
-| Steam boiler | Allows to enable/disable the steam boiler | all |
-| Smart standby enabled | Whether smart standby is on (machine will automatically stand by after given time) | all |
+| Switch name | Description | Available for machines | Controllable through |
+|-------------|-------------| ---------------------- | ------|
+| Main        | Allows to turn machines on-/off | all | <iconify-icon inline title="Cloud" icon="material-symbols:cloud"></iconify-icon> <iconify-icon inline title="Bluetooth" icon="material-symbols:bluetooth"></iconify-icon> |
```
Comment (frenck) [inline source/_integrations/lamarzocco.markdown]: Icons should be used like this:

```suggestion
| Main        | Allows to turn machines on-/off | all | {% icon "material-symbols:cloud" %} {% icon "material-symbols:bluetooth" %} |
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/lamarzocco.markdown (hunk 1 above)
Comment (zweckj) [reply source/_integrations/lamarzocco.markdown]: thanks, forgot to set to draft 🤦‍♂️

Comment (zweckj) [reply source/_integrations/lamarzocco.markdown]: how do you give it a title (used for accessibility/hover) in that syntax?

Comment (zweckj) [reply source/_integrations/lamarzocco.markdown]: there's another problem with that  syntax: for `bluetooth` the linter fails that Bluetooth should be written upper case

Comment (frenck) [reply source/_integrations/lamarzocco.markdown]: > how do you give it a title (used for accessibility/hover) in that syntax?

It doesn't support that, but I'm open for help with extending that specific plugin.

> there's another problem with that syntax: for bluetooth the linter fails that Bluetooth should be written upper case


Hmmm that seems hard to solve right now, let's use the extensive HTML syntax for those as an exception.

Comment (zweckj) [reply source/_integrations/lamarzocco.markdown]: done

Comment (zweckj) [reply source/_integrations/lamarzocco.markdown]: also extended the plugin here https://github.com/home-assistant/home-assistant.io/pull/36004

File: source/_integrations/lamarzocco.markdown [hunk 2]
```diff
@@ -67,77 +67,81 @@ Use Bluetooth:
 
 # Available platforms & entities
 
+{% icon "material-symbols:cloud-outline" %} = La Marzocco Cloud
+<iconify-icon inline title="Bluetooth" icon="material-symbols:bluetooth"></iconify-icon> = Bluetooth
+{% icon "material-symbols:wifi" %} = Local network
```
Comment (klaasnicolaas) [inline source/_integrations/lamarzocco.markdown]: Maybe show this as a list?

I like that you use icons to make it clear to users what is supported ❤️

Comment (zweckj) [reply source/_integrations/lamarzocco.markdown]: I thought about that, but didn't want to use too much estate for the legend. I think it's pretty neat as well, just waiting for another PR, so the icons have proper titles e.g. for screen readers.

**Review by frenck — APPROVED** [review]
Thanks, @zweckj 👍 

../Frenck

**Pull request merged**